### ISSUE TITLE
Please port to python2

### ISSUE THREAD

**Issue by mrpaulmanley** [boabdilperez/snake_game#3]
Python3 isn't stable enough for my production installation of snake. Can you please backport?

**Comment by tomammon** [comment]
agreed, I've got snake running at the center of my fortune 50 app. I have 20M users on this service now. Can you please do the backport by this weekend?

**Comment by tomammon** [comment]
bump. 

is anyone working on this project anymore? is anybody there?
